nautobot/ipam/api/serializers.py

```diff
@@ -165,6 +167,50 @@ class Meta:
             "prefix_length": {"read_only": True},
         }
 
+        detail_view_config = {
+            "layout": [
+                {
+                    "Prefix": {
+                        "fields": [
+                            "prefix",
+                            "parent",
+                            "namespace",
+                            "type",
+                            "network",
+                            "broadcast",
+                            "prefix_length",
+                            "ip_version",
+                            "description",
+                            "role",
+                            "vlan",
+                            "location",
+                            "tenant",
+                            "rir",
+                            "date_allocated",
+                        ]
+                    },
+                },
+            ],
+        }
+
+    @extend_schema_field(serializers.IntegerField)
+    def get_children(self, instance):
+        """
+        Return the child count of a prefix
+        """
+        return instance.descendants_count
+
+    @extend_schema_field(serializers.CharField)
+    def get_utilization(self, instance):
+        """
+        Return the child count of a prefix
+        """
+        utilization_data = instance.get_utilization()
+        numerator = utilization_data[0]
+        denominator = utilization_data[1]
+        utilization = round((numerator / denominator * 100), 1)
+        return str(utilization) + " %"
```

Comment:
How's the performance impact of these fields on the REST API at scale? Should we make them opt-in fields instead of always including them?

Reply:
I didnt realize performance was the reason we excluded them from the story, so I removed those two fields.